onnxruntime/core/providers/cpu/generator/range.cc

```diff
@@ -80,24 +81,43 @@ static Status ComputeRange(OpKernelContext* ctx) {
   return Status::OK();
 }
 
+namespace range_internal {
+
+// Workaround GCC bug https://gcc.gnu.org/bugzilla/show_bug.cgi?id=47226
+// can not specify lambda directly
+template <class T, class... Args>
+inline int invoke_computerange_callable(size_t& called, Status& s, int32_t dt_type, Args&&... args) {
```

Comment:
>invoke_computerange_callable [](start = 11, length = 28)

InvokeComputeRangeCallable

Reply:
Same comment in all the other places where I think you've moved what was a lambda to a function without adjusting the naming to conform to the coding standards.

---
In reply to: [341920627](https://github.com/microsoft/onnxruntime/pull/2307#discussion_r341920627) [](ancestors = 341920627)